doc/sdk-cli-reference.md

```diff
@@ -493,6 +493,8 @@ Runs the tests locally
 * `--local-operator-flags` string - flags that the operator needs, while using --up-local (e.g. \"--flag1 value1 --flag2=value2\")
 * `--no-setup` - disable test resource creation
 * `--image` string - use a different operator image from the one specified in the namespaced manifest
+* --operator-namespace string     Namespace where the operator will be deployed, CRs will be created and tests will be executed (By default it will be in the default namespace defined in the kubeconfig)
```

Comment:
FYI this file is about to be removed. https://github.com/operator-framework/operator-sdk/pull/2682

Reply:
copy that. :+1:
 
shall i leave it there, or do you want me to remove it.

Reply:
@asmacdo i have rebased again.